ocaml/xapi/cli_operations.ml

```diff
@@ -2843,15 +2843,34 @@ let vm_migrate printer rpc session_id params =
              let gpu_group = Client.GPU_group.get_by_uuid remote_rpc remote_session gpu_group_uuid in
              vgpu,gpu_group) (read_map_params "vgpu" params) in
 
-         let default_sr =
-           try let pools = Client.Pool.get_all remote_rpc remote_session in
-             printer (Cli_printer.PMsg "Selecting remote pool's default SR for migrating VDIs") ;
-             Some (Client.Pool.get_default_SR remote_rpc remote_session (List.hd pools))
+         let preferred_sr =
+           try
+             let query = Printf.sprintf "(field \"host\"=\"%s\") and (field \"currently_attached\"=\"true\")" (Ref.string_of host) in
```

Comment:
If you want to avoid escaping inside the string you can use `{|(field "host"...|}` (see https://caml.inria.fr/pub/docs/manual-ocaml/extn.html#sec263, this is a new but useful language feature in OCaml 4.02), but is fine as it is too.